pyproject.toml

```diff
@@ -2,6 +2,7 @@
 
 include = [
     "miniwob/**",
+    "tests/**",
```

Comment:
I don't think we should be including `tests` in the release

Reply:
Sorry I thought this line was for the files that are included in the release.
You are correct, this is the correct way of adding tests to pyright.
Though I should note that we normally don't care about having correct type hints for the tests

Reply:
Thanks. It still helps with catching mistakes like missing method arguments, so I would prefer to keep it here for now.